stock-filters/ChirpVillage/Surface.py

```diff
@@ -26,8 +26,31 @@ def __init__(self, level, box):
         self.z_length = abs(self.z_end - self.z_start)
         self.surface_map = self.init_surface_map(level)
         self.surface_map = self.calc_steepness()
+        self.surface_map = self.find_waterways(level)
         self.door_blocks = []
 
+    def find_waterways(self, level):
```

Comment:
can this be changed to find lava and water. just water is misleading as it does both